src/handlers.js

```diff
@@ -110,20 +110,28 @@ async function homePage(req, res, timecop) {
   if (cached !== null) {
     timecop.end("cache-check");
     // We know our cache is good and lets serve the data
-    res.render("home", { featured: cached, timecop: timecop.timetable, page: homePage });
+    res.render("home", { ...cached, timecop: timecop.timetable, page: homePage });
   } else {
     // the cache is invalid.
     timecop.end("cache-check");
     timecop.start("api-request");
     try {
-      let api = await superagent.get(`${apiurl}/api/packages/featured`);
+      let featured = await Promise.all([
+        superagent.get(`${apiurl}/api/packages/featured`),
+        superagent.get(`${apiurl}/api/themes/featured`)
+      ]).then(([featuredPackages, featuredThemes]) => Promise.all([
+        utils.prepareForListing(featuredPackages.body),
+        utils.prepareForListing(featuredThemes.body)
+      ])).then(([featuredPackages, featuredThemes]) => ({
+        featuredPackages,
+        featuredThemes
+      }));
       timecop.end("api-request");
       timecop.start("transcribe-json");
-      let obj = await utils.prepareForListing(api.body);
```

Comment:
Why was this removed?

Reply:
@CatPerson136 I appear to not be able to copy line references on the iPhone app, but it was deleted and moved to line 123

Reply:
Updated!